### PR TITLE
Set up sql database

### PR DESCRIPTION
**Pull request by williambaibai**
Add the skeleton for SQL database

### CODE REVIEW

File: src/app/models/user.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import *
```
Comment (cswenberg) [inline src/app/models/user.py]: what is this?

File: src/config.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import os
+
+# Ensure Python path
+basedir = os.path.abspath(os.path.dirname(__file__))
```
Comment (cswenberg) [inline src/config.py]: snake case

File: src/app/__init__.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Configure Flask app
+app = Flask(__name__, static_url_path='/static')
+app.config.from_object(os.environ['APP_SETTINGS'])
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
+
+# Database
+db = SQLAlchemy(app)
+
+# Import + Register Blueprints
+blueprint = Blueprint('controllers', __name__, url_prefix='/api/v0')
```
Comment (meganle) [inline src/app/__init__.py]: I think we should do 'v1' first. I think that's what the other backends start with

**Comment by meganle** [comment]
@cswenberg I believe the plan is to still serve gym data through GraphQL, but future features will require users, other entities, etc. which will involve a DB

File: src/app/__init__.py (hunk 3 above)
Comment (williambaibai) [reply src/app/__init__.py]: true, good point

File: src/app/models/user.py (hunk 1 above)
Comment (williambaibai) [reply src/app/models/user.py]: this is just the import statement for the user model which I have yet started writing

**Pull request merged**